pages/linux/debtap.md

```diff
@@ -0,0 +1,21 @@
+# debtap
+
+> Convert Debian packages into Arch Linux packages.
+> See also: `pacman-upgrade`.
+> More information: <https://github.com/helixarch/debtap>.
+
+- Update debtap database (before first run):
+
+`sudo debtap --update`
+
+- Convert the specified package:
+
+`debtap {{path/to/package.deb}}`
+
+- Convert the specified package (bypass all questions, not recommended):
```

Comment:
```suggestion
- Convert the specified package bypassing all questions, except for editing metadata files:
```

Reply:
Following the contributing guidelines, I think it's better to keep it short and simple 😃

Reply:
For convenience, just skip them all and install to the local machine later on.